db/seeds.rb

```diff
@@ -135,6 +137,7 @@
 
 p 'assigning worksheets to work groups'
 
+work_groups = WorkGroup.all
 work_groups.each_with_index do |work_group, idx|
```

Comment:
needs to redo this part too
the logic was implemented for when there were only 2 classrooms, so it uses only 2 different templates belonging to 2 teacher
now each class needs to pull from their own worksheet template

Reply:
Done!